server/src/main/java/org/apache/druid/query/lookup/LookupSerdeModule.java

```diff
@@ -58,6 +59,9 @@ public List<? extends Module> getJacksonModules()
   public void configure(Binder binder)
   {
     JsonConfigProvider.bind(binder, LookupModule.PROPERTY_BASE, LookupConfig.class);
+    //Even though LookupSerdeModule will be used for processes that are not loading lookups, we need to bind LookupNodeService,
+    //so that TaskToolboxFactory and TaskToolbox will get their dependency. Binding it won't load lookups.
```

Comment:
Nit:
```suggestion
    // Even though LookupSerdeModule will be used for processes that are not loading lookups, we need to bind LookupNodeService,
    // so that TaskToolboxFactory and TaskToolbox will get their dependency. Binding it won't load lookups.
```